README.md

```diff
@@ -1 +1,102 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+### Problems
+1. Not being able to merger speeches data with members data
+
+speeches
```

Comment:
You don't need to give so many examples, for simplicity, just show 2 or max 3 results from the speeches api